guides/common/modules/proc_creating-a-playbook-with-modules-from-project-ansible-collection.adoc

```diff
@@ -97,6 +98,24 @@ If you are authenticating with a Kerberos ticket, map the `server_url` parameter
       use_gssapi: true
   tasks:
 ----
++
+If you need to use an HTTP proxy to reach the {Project} API, set the `https_proxy` environment variable:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+- name: _My Playbook_
+  hosts: _localhost_
+  environment:
+    https_proxy: "_http://proxy.example.com:8080_"
```

Comment:
I kinda didn't want to *also* adjust the vault.yml above to include the proxy url… but it can contain sensitive data and probably should live there.

Reply:
Then perhaps instead of lines 101-118, a better idea might be to add a substep to the place where users are populating the vault.